app/gateway/changelog.md

```diff
@@ -3182,6 +3182,77 @@ openid-connect
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
 
+## 2.8.4.5
+**Release Date** 2023/11/28
+
+### Features
+#### Core
+* Added support for observing the time consumed by some components in the given request.
+* A unique Request ID is now populated in the error log, access log, error templates, log serializer, and in a new X-Kong-Request-Id header (configurable for upstream/downstream using the `headers` and `headers_upstream` configuration options).
+
+#### Enterprise
+* **Admin API:** Added counters such as routes, plugins, licenses, and deployment info to the report component. Also, added a checksum and timestamp to the output.
+* Backported the Request ID feature.
+
+#### Plugins
+* [OpenID Connect](/hub/kong-inc/openid-connect/) (`openid-connect`)
+  * New field `unauthorized_destroy_session`, which when set to true, it destroys the session (delete the user's session cookie) when the request is unauthorized. Default is set to true. Set to false to preserve the session.
+
+### Fixes
+#### Core
+* Dismissed confusing debug log from the Redis rate limiting tool.
+* Removed the asynchronous timer in `syncQuery()` to prevent hang risk.
+* Updated the DNS client to follow configured timeouts in a more predictable manner.
+* Ensured pluginserver protobuf includes are placed in the correct path in packages.
+* Added missing support for consumer group tags.
+* Fixed an issue that caused Kong Gateway to fail to start if `proxy_access_log` is `off`.
+* Removed asynchronous timer in `syncQuery()` to prevent hang risk.
+* Fixed an issue that called `store_connection` without passing `self`.
+* Now Kong Gateway uses deep copies of route, service, and consumer objects when log serialize.
+* Added per request debugging.
+* Fixed an issue that caused a failure to broadcast keyring material when using the cluster strategy.
+* Addressed a problem where an abnormal socket connection would be reused when querying the PostgreSQL database.
+
+#### Plugins
```

Comment:
I added the keyring fix and one of the plugin server fixes. I omitted https://github.com/Kong/kong-ee/pull/6906 because it was removing unused code, which I didn't think warranted a release note since it wasn't a bug, just a backend code change.